roles/st2web/tasks/certificate.yml

```diff
@@ -0,0 +1,37 @@
+---
+- name: Verify if custom SSL certificate was correctly specified
+  fail:
+    msg: "When using custom certificate, both 'st2web_ssl_certificate' and 'st2web_ssl_certificate_key' must be provided"
+  # no XOR in Yaml
+  when: (st2web_ssl_certificate and not st2web_ssl_certificate_key) or (not st2web_ssl_certificate and st2web_ssl_certificate_key)
+
+- name: Create SSL certificate directory
+  become: yes
+  file:
+    state: directory
+    dest: /etc/ssl/st2
+    mode: 0700
+
+- name: Save custom SSL certificate
```

Comment:
@armab This would all be simple if ansible-st2 would allow galaxy imports so we could just use jdauphant's SSL role.